lib/index.js

```diff
@@ -65,6 +65,7 @@
             console.log('Error Message:   ' + item.errMsg);
             console.log('Rule ID:         ' + item.ruleID);
             console.log('Severity:        ' + item.severityEnum);
+            console.log('Result:          ' + item.result);
```

Comment:
Good point. Currently both ERROR's and WARNING's are logged as result = false which is a failure. Only severityEnum.INFO is presumed to be a SUCCESS. May be move it to an enhancement ?